minecraft_servers/phoenix/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "phoenix",
+  "nice_name": "Phoenix",
+  "direct_ip": "rainbowfurry.com",
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://rainbowfurry.com",
+    "web_shop": "https://phoenix-network-store.tebex.io"
```

Comment:
Unavailable

Reply:
I removed the webstore for now. we have some problems on it for the moment but will add it later on